src/BSON/ObjectId.stub.php

```diff
@@ -15,7 +15,7 @@ public final function getTimestamp(): int {}
 
     public final function __toString(): string {}
 
-    public static function __set_state(array $properties): ObjectId {}
+    public final static function __set_state(array $properties): ObjectId {}
```

Comment:
For consistency, please place the `final` keyword _before_ the method visibility (e.g. `final public static`). I realize many of the stubs are inconsistent about this. I've opened [PHPC-2136](https://jira.mongodb.org/browse/PHPC-2136) to address that.

Reply:
I've removed configure~ and add final to other __set_state() methods and by the way moving final keyword before the method visibility where I noted it but still there are many places where order is not correct

Reply:
Also after generation stubs I noticed that order for key words does not any affect to generated files.

this is the code fragment that add arg infos
`private function getFlagsAsArginfoString(): string
    {
        $flags = "ZEND_ACC_PUBLIC";
        if ($this->flags & Class_::MODIFIER_PROTECTED) {
            $flags = "ZEND_ACC_PROTECTED";
        } elseif ($this->flags & Class_::MODIFIER_PRIVATE) {
            $flags = "ZEND_ACC_PRIVATE";
        }

        if ($this->flags & Class_::MODIFIER_STATIC) {
            $flags .= "|ZEND_ACC_STATIC";
        }

        if ($this->flags & Class_::MODIFIER_FINAL) {
            $flags .= "|ZEND_ACC_FINAL";
        }

        if ($this->flags & Class_::MODIFIER_ABSTRACT) {
            $flags .= "|ZEND_ACC_ABSTRACT";
        }

        if ($this->isDeprecated) {
            $flags .= "|ZEND_ACC_DEPRECATED";
        }

        return $flags;
    }`

Reply:
> there are many places where order is not correct

Indeed, this is why I opened [PHPC-2136](https://jira.mongodb.org/browse/PHPC-2136). That can be addressed independently from this PR.

The order of modifiers has no impact on the generated C code or evaluation by PHP. I just want this addressed so the prototypes throughout our stubs are consistent.

Reply:
If I understand correct there is no more changes. Can we merge this PR and continue work on PHPC-2136 after next triage